### PR TITLE
Update docs image paths and remove outdated images

### PR DESCRIPTION
**Pull request by discdiver**
Removes clearly outdated images and fixes image paths.

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
 	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] If this pull request adds new functionality, it includes unit tests that cover the changes
- [X] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `mint.json` for files that are removed or renamed.

For new functions or classes in the Python SDK:

- [ ] This pull request includes helpful docstrings.
- [ ] If a new Python file was added, this pull request contains a stub page in the Python SDK docs and an entry in `docs/mint.json` navigation.

### CODE REVIEW

File: docs/3.0rc/concepts/artifacts.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ Artifacts are persisted outputs such as tables, Markdown, or links.
 They are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI.
 Artifacts make it easy to track and monitor the objects that your flows produce and update over time.
 
-![Markdown artifact sales report screenshot](/img/ui/md-artifact-info.png)
+![Markdown artifact sales report screenshot](/3.0rc/3.0rc/img/ui/md-artifact-info.png)
```
Comment (billpalombi) [inline docs/3.0rc/concepts/artifacts.mdx:10]: Is this right?

**Review by billpalombi — COMMENTED** [review]
I may have caught a duplicate. Otherwise looks good

File: docs/3.0rc/concepts/artifacts.mdx (hunk 1 above)
Comment (discdiver) [reply docs/3.0rc/concepts/artifacts.mdx:10]: good catch. Tested that manually.

**Review by billpalombi — APPROVED** [review]
lgtm

**Pull request merged**